app/javascript/pages/browse/TileListItem.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <li class="tileBrowser-listItem box">
+    <div class="tileBrowser-listItem-categories">
+      <TagList :tags="category_tags" tagClasses="tag is-info is-light is-inline-flex" />
+    </div>
+    <div class="tileBrowser-listItem-body">
+      <TagList v-if="service_area" :tags="[service_area]" />
+      <h5 class="subtitle">{{ short_title }}</h5>
+      <p>{{ description }}</p>
+      <div>
+        <time :datetime="publish_until"></time>
+      </div>
+    </div>
+    <div class="tileBrowser-listItem-actions">
+      <button class="button icon-list is-primary is-outlined">View Profile</button>
+      <button class="button icon-list is-primary is-outlined">Contact</button>
+    </div>
```

Comment:
I was not aware of that. It looks like it doesn't help here at the tablet size, though, unless I've got some other way to also defend against overflow:

<img width="678" alt="Screen Shot 2020-05-02 at 6 17 02 PM" src="https://user-images.githubusercontent.com/3620291/80893544-40064f00-8ca1-11ea-8ce2-8e452b09e921.png">

Reply:
@h-m-m we may want to go w the profile icon (like in the pictures for the admin view) rather than `View Profile` at that size?

Reply:
Also, bc we still have to define what happens after clicking on a Contribution, I'd last landed at `Respond` rather than `Contact` in case we aren't definitely directly connecting people (in the case of a peer-meditated match).